plugins/chat/db/migrate/20221101061319_add_last_editor_id_to_chat_messages.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddLastEditorIdToChatMessages < ActiveRecord::Migration[7.0]
+  def change
+    add_column :chat_messages, :last_editor_id, :integer
+    add_column :chat_message_revisions, :user_id, :integer
```

Comment:
Should we make a followup PR to prevent these columns to ever be null?

Reply:
Yes good idea, I will do that too :+1: